src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss

```diff
@@ -9,6 +9,19 @@
     > div{
         padding: 0;
     }
+    #wi-comment-add-comment {
+        /* Overrides pf's form-control style's height property */
+        /* The min-height is needed to avoid zero-height element */
+        /* The min-height is same as form-control's default height */
+        height: auto;
+        min-height: 26px;
```

Comment:
I went with the BS/PF default for text-input (and they go with px).

Need to check with the UXD on this. With textarea, the field used to be 3-rows in height, not sure that was by default or as per UXD definition.